### PR TITLE
feat: add struct example with Person

### PR DESCRIPTION
**Pull request by dghosef**

### CODE REVIEW

File: examples/struct-person.flix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+struct Person[r] {
+    name: String,
+    age: Int32,
+    height: Int32
+}
+
+def giveBirth(name: String, rc: Region[r]): Person[r] \ r =
+    new Person { name = name, age = 0, height = 30 } @ rc
+
+def displayPerson(person: Person[r]): Unit \ {r, IO} =
```
Comment (magnus-madsen) [inline examples/struct-person.flix:11]: Lets drop this and simply inline the print into main.

Comment (magnus-madsen) [reply examples/struct-person.flix:11]: You can also use string interpolation "foo${person.blarg}"to make it smaller.

**Review by magnus-madsen — COMMENTED** [review]
I like the example. Let us polish it up: Add some comments that gives a flow.

See the other examples on https://flix.dev/

File: examples/struct-person.flix (hunk 1 above)
Comment (dghosef) [reply examples/struct-person.flix:11]: Sorry, did you mean to drop `giveBirth` or `displayPerson`?

**Comment by dghosef** [comment]
!!rerun

**Review by magnus-madsen — APPROVED** [review]

**Pull request merged**